lib/ProductOpener/Tags.pm

```diff
@@ -5245,6 +5247,43 @@ sub get_knowledge_content ($tagtype, $tagid, $target_lc, $target_cc) {
 	return;
 }
 
+=head2 create_property_to_tag_mapping_table ($tagtype, $property)
+
+Given a tag type and a property name, create a mapping table from the property to the tag id.
+
+=head3 Arguments
+
+=head4 $tagtype
+
+The type of the tag (e.g. categories, labels, allergens)
+
+=head4 $property
+
+The property name (e.g. "gpc_category_code:en:")
+
+=head3 Return value
+
+A hash reference with the tag id as key and the property value as value.
+
+=cut
+
+sub create_property_to_tag_mapping_table ($tagtype, $property) {
+	my %mapping = ();
+
+	if (exists $properties{$tagtype}) {
+		foreach my $tagid (keys %{$properties{$tagtype}}) {
+			my $value = $properties{$tagtype}{$tagid}{$property};
+			if (defined $value) {
+				$value =~ s/^\s+//;
+				$value =~ s/\s+$//;
```

Comment:
why do we do that ? A comment is welcome.

I don't think the second line is useful. Or should the first line be `value =~ s/^\s+/\s/;` ?

Reply:
I think we can remove both lines in fact